glyphs2fontir/src/source.rs

```diff
@@ -548,8 +548,13 @@ impl Work<Context, WorkError> for KerningWork {
                 glyph
                     .left_kern
                     .iter()
-                    .map(|group| (KernSide::L, group))
-                    .chain(glyph.right_kern.iter().map(|group| (KernSide::R, group)))
+                    .map(|group| (KernSide::Side1, group))
```

Comment:
I commented on the meaning of left/right in kernLeft/kernRight glyph attributes here: https://github.com/googlefonts/fontc/pull/331#discussion_r1223305938

I believe you have confused the two here. You are associating `left_kern` (which you renamed `kern_left`  in #331) to `KernSide::Side1`, but actually `kernLeft` (`leftKerningGroup` in Glyphs v2 terminology) is the group of glyphs that have the same kerning towards their left, so for LTR kerning that means logical side2, not side1...

Reply:
```python
    if side == 1:
        return "rightKerningGroup"
    else:
        return "leftKerningGroup"
```

https://github.com/googlefonts/glyphsLib/blob/42bc1db912fd4b66f130fb3bdc63a0c1e774eb38/Lib/glyphsLib/builder/groups.py#L192-L195

as used in 

https://github.com/googlefonts/glyphsLib/blob/42bc1db912fd4b66f130fb3bdc63a0c1e774eb38/Lib/glyphsLib/builder/groups.py#L115-L123

Reply:
Ugh indeed I misunderstood and didn't take it as "toward their left" but rather "goes on the left. Will fix, ty.

Reply:
I will copy https://github.com/googlefonts/fontc/compare/kern3...kern4?expand=1 in and update the glyphs2 copy.

Reply:
Take 2 is posted.